releases/release-1.23/release-notes/maps/pr-101370-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 101370
+releasenote:
+  text: Create HPA with follow：
```

Comment:
This seems like an incomplete release note. @cici37 Any thoughts on how can we verify this ?

Reply:
Good catch! An easy way to do a sample verification is through the pr number. It seems like an issue with commit scanning. This PR actually should not have a release note. @parul5sahoo Would you mind to delete this one?
@wilsonehusin Here might be an krel issue.

Reply:
this is intended behavior actually -- the author puts out the release note `NONE`, but there was no codeblock identifier, therefore krel picks up the first codeblock.

for reference, if you look at the PR 101370, the author puts the release note as such:

```
    ```
    NONE
    ```
```

however, if you create a new pull request in k/k, the template is actually something like this:

```
    ```releasenote
    Release note here
    ```
```

in summary, krel prioritizes codeblocks with `releasenote` annotation, and when none found, krel will use the first codeblock available.

the quickest fix is to add `do_not_publish: true` in this map, or we can leave the note to PR author to add `releasenote` annotation 😄

Reply:
example of using `do_not_publish`:
https://github.com/kubernetes/sig-release/blob/03b88acdfd682f2eeb973b7dd5389b4d3f227521/releases/release-1.22/release-notes/maps/pr-100262-map.yaml#L1-L3